app/views/shared/_interactions_tab.html.haml

```diff
@@ -0,0 +1,18 @@
+/view for interactions tab for gene/drug
+/lists interaction summary panels
+
+- if defined? drug
+	- entity = drug 
```

Comment:
The indentation in this file is off (too far indented). The standard is two spaces per indentation level.